cdm/s3/src/test/java/thredds/inventory/s3/TestMFileS3.java

```diff
@@ -108,7 +108,7 @@ public void bucketAndKeyGcs() throws IOException {
   @Test
   @Category(NotPullRequest.class)
   public void bucketAndKeyOsdc() throws IOException {
-    long lastModified = 1611593614000L;
+    long lastModified = 1619161328000L;
```

Comment:
this tests is still going through the same codepath (i.e. getting last modified from headObjectResponse) but the lastmodified seems to have changed

Reply:
Yeah, I think we should disable the last modified portion of the tests. It seems like the last modified time changes occasionally on all platforms, but I'm not sure why...maybe shifting disks on the object store or data centers? It has happened most often on the Open Science Data Cloud.

Reply:
I can do that